common/Makefile.am

```diff
@@ -24,13 +28,15 @@ libyami_common_cppflags = \
 	$(NULL)
 
 lib_LTLIBRARIES	              = libyami_common.la
-libyami_commonincludedir      = ${includedir}/libyami_common
+libyami_commonincludedir      = ${includedir}/libyami
 libyami_commoninclude_HEADERS = $(libyami_common_source_h)
+noinst_HEADERS                = $(libyami_common_source_h_priv)
 libyami_common_la_SOURCES     = $(libyami_common_source_c)
 libyami_common_la_LDFLAGS     = $(libyami_common_ldflags)
 libyami_common_la_CPPFLAGS    = $(libyami_common_cppflags)
 
 DISTCLEANFILES = \
-	Makefile.in 
-
+	Makefile.in \
+	YamiVersion.h.in \
```

Comment:
This should be YamiVersion.h

We don't want to delete the YamiVersion.h.in file.